slackbridge/messages.py

```diff
@@ -71,19 +83,54 @@ def resolve(self):
                 user_bot.leave(channel_name)
             return
 
+    def is_bot_user(self):
+        """Sometimes bot_id is not included and other
+        times it is passed as None. Checks both cases."""
+        return ('bot_id' in self.raw_message and
+                self.raw_message['bot_id'] is not None)
+
     def _change_presence(self, user_bot):
         if self.raw_message['presence'] == 'away':
             user_bot.away('Slack user inactive.')
         elif self.raw_message['presence'] == 'active':
             user_bot.back()
 
-    def _irc_me_action(self, channel_name, user_bot):
+    def _irc_me_action(self, channel_name, user_bot, action):
         user_bot.post_to_irc(
             user_bot.describe,
             '#' + channel_name,
-            self.raw_message['text'],
+            action,
         )
 
+    # TODO: Change image host to fluffy.cc in the future?
+    def _post_to_imgur(self, channel_name, user_bot, file_data):
+        ext = file_data['filetype']
+        if ext not in ALLOWED_FILE_EXTS:
+            return
+
+        auth = {'Authorization': 'Bearer {}'.format(
+            self.bridge_bot.slack_token)}
+        r = requests.get(file_data['url_private'],
+                         headers=auth, stream=True)
+        if r.status_code != 200:
+            log.msg('Could not GET image from: {}'.format(
+                file_data['url_private']))
```

Comment:
I'll only mention this once here, but it applies to all the wrapped lines in this PR. I'd format anything that wraps lines like this like the following so that it's easier to add more parameters and looks a bit cleaner and clearer as to where the statement actually ends:

```python
log.msg('Could not GET image from: {}'.format(
    file_data['url_private'],
))
```

Not a big deal, but I think it makes it a bit cleaner.